client/src/main/scala/org/apache/celeborn/client/write/LifecycleManager.scala

```diff
@@ -15,7 +15,7 @@
  * limitations under the License.
  */
 
-package com.aliyun.emr.rss.client.write
+package org.apache.celeborn.client.write
```

Comment:
Moving it to org.apache.celeborn.client will be better.

Reply:
We can do it later. This pr merely changes package names.